documentation/index.md

```diff
@@ -1,28 +1,31 @@
-<!--
-SPDX-FileCopyrightText: 2023 Andreas Kahler <mail@andreaskahler.com>
+# Nimble Assembly Guide
 
-SPDX-License-Identifier: CERN-OHL-S-2.0
--->
 
-# Nimble Rack Assembly
+## What is Nimble
```

Comment:
None of us use consistent capitalization on "nimble" vs "Nimble".

Reply:
Moved this discussion to #67